src/libs/actions/IOU.js

```diff
@@ -178,6 +178,13 @@ function setMoneyRequestAmount_temporaryForRefactor(transactionID, amount, curre
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {amount, currency});
 }
 
+/**
+ * @param {String} transactionID
```

Comment:
Description:

Reset the money request amount, discarding the user-provided value. In the case of distance requests, this will effectively re-enable the default behavior of automatic amount calculation.

Reply:
updated